### ISSUE TITLE
Throwing error

### ISSUE THREAD

**Issue by parthi2929** [takluyver/bookbook#17]
Hi

I tried using your bookbook, but I am getting error as below. could not understand what is going wrong. I had simplified my file names as required. 

![image](https://user-images.githubusercontent.com/11896263/49022232-5bbb1b00-f1ba-11e8-9d4b-2c2d9aac2ba2.png)

Any idea why this appear? Below is my current directory from where I run the command. 
![image](https://user-images.githubusercontent.com/11896263/49022265-6fff1800-f1ba-11e8-9548-9124b884acfe.png)

**Comment by takluyver** [comment]
Bookbook expects filenames beginning with a number and a dash, like `1-intro.ipynb`. The error message could be clearer about that.

**Comment by parthi2929** [comment]
Ok It already has "121 - xxx". I had 121 for a reason. The space in between was a problem?

**Comment by takluyver** [comment]
Yup, it expects it without a space. The requirement could be relaxed, it's just a fairly quick implementation.
